dna/dna.json

```diff
@@ -1,172 +1,196 @@
 {
-    "Version": 0,
-    "UUID": "00000000-0000-0000-0000-000000000000",
-    "Name": "clutter",
-    "RequiresVersion": 21,
-    "Properties": {
-        "description": "distributed micro-blogging (Twitter clone)",
-        "language": "en",
-        "enableDirectoryAccess" : "true"
-    },
-    "PropertiesSchemaFile": "properties_schema.json",
-    "DHTConfig": {
-        "HashType": "sha2-256"
-    },
-    "BasedOn": {
-        "H": null
+  "Version": 0,
+  "UUID": "00000000-0000-0000-0000-000000000000",
+  "Name": "clutter",
+  "RequiresVersion": 21,
+  "Properties": {
+    "description": "distributed micro-blogging (Twitter clone)",
+    "language": "en",
+    "enableDirectoryAccess": "true"
+  },
+  "PropertiesSchemaFile": "properties_schema.json",
+  "DHTConfig": {
+    "HashType": "sha2-256"
+  },
+  "BasedOn": {
+    "H": null
+  },
```

Comment:
With the latest head in https://github.com/holochain/holochain-proto develop branch, this
``
"BasedOn": {
  "H": null
}
``
makes it fail.

Can you try running this version against the latest Holochain version, please, @science-girl?

I've removed these lines locally which enables Holochain to start but then I only see a
``
404 page not found
``

Reply:
Can you provide more background on this? I don't know what purpose "BasedOn" serves and how I can fix this.